docs/wiki/research/roadmap.md

```diff
@@ -74,74 +74,100 @@ Upgrades related to censorship resistance, decentralization and mitigating proto
 | Distributed Block Building         | MEV-Track | Endgame Block Production Pipeline |                                               Decentralize the block building process, by distributing it                                                |                                                                              Decentralize different parts of the Builder: </br> (i) the algorithms for choosing transactions (the block building transaction ordering) </br> (ii) resources for block construction, especially under full Danksharding (split-up big blocks)  </br> (iii) add extra builder services (e.g.Preconfirmations)                                                                              | in research </br> [Preconfirmations](/wiki/research/Preconfirmations/Preconfirmations.md),</br> SUAVE[^25] |
 | Application Layer MEV Minimization | MEV-Track |                 -                 |                                                         App layer effort to minimize harmful MEV                                                         |                                                                                                                                                                                        The minimization techniques target: </br>(i) frontrunning, and </br>(ii) sandwich attacks                                                                                                                                                                                         | Examples[^26]                                                                                              |
 | Preconfirmations                   | MEV-Track |                 -                 |                       Users preconfirmations on transaction execution, for a competitive user experience in Ethereum interactions                        |                                                                                                                                  Block builders could publicly agree to include transactions with a priority fee over a certain amount, and send users a receipt indicating their intent to include the transaction in a specific block                                                                                                                                  | [in research](/wiki/research/Preconfirmations/Preconfirmations.md)[^27]                                    |
+      
+Here is a summarized table of the provided content for the Staking Economics/Experiences Track:
```

Comment:
I don't really understand what is the framing of this table compared to the one above. Could it be merged? I would just add a bit more explanation to provide context to these extra topics

Reply:
@raxhvl and @taxmeifyoucan I just made the requested changes now. please let me know if its okay.

Reply:
> I don't really understand what is the framing of this table compared to the one above. Could it be merged? I would just add a bit more explanation to provide context to these extra topics

I agree

> I don't really understand what is the framing of this table compared to the one above. Could it be merged? I would just add a bit more explanation to provide context to these extra topics

I agree with @taxmeifyoucan here, @gconnect. MAX_EB line entry is OK, the other ones should follow that and the above formatting: first column should be the proposed roadmap Upgrade (e.g. ePBS), the second column is the roadmap track(this is correctly added) and the third column represents the topic (which you passed in the first column, actually)

Reply:
> I don't really understand what is the framing of this table compared to the one above. Could it be merged? I would just add a bit more explanation to provide context to these extra topics

I agree with @taxmeifyoucan here, @gconnect. MAX_EB line entry is OK, the other ones should follow that and the above formatting: first column should be the proposed roadmap Upgrade (e.g. ePBS), the second column is the roadmap track(this is correctly added) and the third column represents the topic (which you passed in the first column, actually)

Reply:
> > I don't really understand what is the framing of this table compared to the one above. Could it be merged? I would just add a bit more explanation to provide context to these extra topics
> 
> I agree with @taxmeifyoucan, @gconnect. MAX_EB line entry is OK, the other ones should follow that, and the above formatting: first column should be the proposed roadmap Upgrade itself(e.g. ePBS), the second column is the roadmap track(this is correctly added) and the third column represents the topic (which you actually passed in the first column)

I just made some adjustment. Please take a look and let me know what you think